ui/page/proposal/proposals_page.go

```diff
@@ -527,8 +527,10 @@ func (pg *ProposalsPage) layoutAuthorAndDate(gtx C, item proposalItem) D {
 }
 
 func (pg *ProposalsPage) layoutInfoTooltip(gtx C, rect image.Rectangle, item proposalItem) {
-	inset := layout.Inset{Top: values.MarginPadding20, Left: values.MarginPaddingMinus230}
+	inset := layout.Inset{Top: values.MarginPadding20, Left: unit.Dp(-195)}
```

Comment:
Move this to the dimensions.go file.

```suggestion
	inset := layout.Inset{Top: values.MarginPadding20, Left: values.MarginPaddingMinus195}
```